2/Dockerfile.rhel7

```diff
@@ -57,7 +57,7 @@ COPY ./contrib/jenkins /usr/local/bin
 ADD ./contrib/s2i /usr/libexec/s2i
 ADD release.version /tmp/release.version
 
-RUN /usr/local/bin/install-jenkins-core-plugins.sh /opt/openshift/base-plugins.txt && \
+RUN /usr/local/bin/install-jenkins-core-plugins.sh /opt/openshift/base-plugins.txt " " && \
```

Comment:
would be cleaner to make the script tolerate the absence of the second arg.

Reply:
(shift would probably get you there: https://www.tldp.org/LDP/Bash-Beginners-Guide/html/sect_09_07.html)